src/Components/Switcher/index.tsx

```diff
@@ -8,8 +8,10 @@ import './Switcher.scss'
 export type Props = {
   /** Define theme */
   theme?: ETheme
+  /** Custom class */
+  className?: string
   /** To set the target */
-  id: string
+  // id: string
```

Comment:
Why did you comment "id" field? I think a user habe possibility to add id-key for this component